utils/sprite.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, sprite, effect=None, reaction=None):
 
     @property
     def center(self):
-        return int(self.sprite().x), int(self.sprite().y)
+        return self.sprite().x, self.sprite().y
```

Comment:
@mtschaffer you should make sure this doesn't introduce errors in collision detection